src/Plugin.Maui.Audio/Plugin.Maui.Audio.csproj

```diff
@@ -11,7 +11,7 @@
 
 		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">14.2</SupportedOSPlatformVersion>
 		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'maccatalyst'">14.0</SupportedOSPlatformVersion>
-		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">21.0</SupportedOSPlatformVersion>
+		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">23.0</SupportedOSPlatformVersion>
```

Comment:
I assume this was intended? Are you using APIs that are only available in API 23? If at all possible I'd like to keep this on 21 so that we support the same version as .NET MAUI itself.

Reply:
For the plugin, please see my previous reply. For the sample application, it was an result on the change on the Plugin.